server/protocols/jmap-integration-testing/memory-jmap-integration-testing/src/test/resources/listeners.xml

```diff
@@ -24,5 +24,10 @@
   </listener>
   <listener>
     <class>org.apache.james.mailbox.quota.mailing.listeners.QuotaThresholdCrossingListener</class>
+    <configuration>
+      <thresholds>
+        <threshold>0.1</threshold>
+      </thresholds>
+    </configuration>
```

Comment:
can we have a sample with 2 listeners with different configurations?

Reply:
Hehe it messes up as the same event store is used!!!

One of the listeners resets the thresholds of the other one.

We should forbid registration of several `QuotaThresholdCrossingListener`. Maybe in another task (as that's quite a big diff).

We could introduce MailbixListenersConfigurationValidator, then inject one for Quota mailing that would check we get at most one listener of that type. WDYT?

Reply:
IMO, the problem is not that we use the same EventStore for both listener, it's that we use the same aggregateId pattern. When we use more than a single QuotaThresholdCrossingListener, we have to name it.
I'll give it a quick try.